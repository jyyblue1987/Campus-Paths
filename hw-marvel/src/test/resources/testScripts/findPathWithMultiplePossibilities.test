# check if the output is correct for finding the shortest path when multiple maths are possible

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4

AddEdge g n1 n2 a
AddEdge g n2 n1 b
AddEdge g n4 n1 c
AddEdge g n3 n2 d
AddEdge g n2 n3 e
AddEdge g n3 n2 f
AddEdge g n1 n4 h
AddEdge g n4 n2 i

FindPath g n1 n3