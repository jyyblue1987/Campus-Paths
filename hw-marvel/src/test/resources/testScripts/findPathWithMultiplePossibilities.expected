# check if the output is correct for finding the shortest path when multiple maths are possible

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g

added edge a from n1 to n2 in g
added edge b from n2 to n1 in g
added edge c from n4 to n1 in g
added edge d from n3 to n2 in g
added edge e from n2 to n3 in g
added edge f from n3 to n2 in g
added edge h from n1 to n4 in g
added edge i from n4 to n2 in g

path from n1 to n3:
n1 to n2 via a
n2 to n3 via e